### ISSUE TITLE
Way to get List/Library Item Version information

### ISSUE THREAD

**Issue by rahulsuryawanshi** [SharePoint/PnP-JS-Core#622]
#### Category
[X ] Enhancement

[ ] Bug

[ X] Question

#### Version

Please specify what version of the library you are using: [  3.0.1  ]

_If you are not using the latest release, please update and see if the issue is resolved before submitting an issue._

#### Expected / Desired Behavior / Question
I am trying to find way to get list item version information. I do see that there is REST API available to get version of particular list item with /_api/web/lists/getbyTitle('<List Title>')/items/getById('1')/versions

I did not find method with Pnp-JS API which can give me version information for particular list item. 

I have read all documentation and didn't find any reference about it. I am not sure if I am missing about how to find it.

Thanks!

**Comment by koltyakov** [comment]
Hey @rahulsuryawanshi, 

Item's versions can be requested like this:

```TypeScript
pnp.sp.web.getList('/sites/sile/lists/list')
	.items.getById(2)
	.select('Versions')
	.expand('Versions')
	.get().then(console.log)
```

Yet, it returns quite limited information.

`/_api/web/getList('/sites/sile/lists/list')/items/getById(1)/versions` endpoint looks like is not implemented for items, only for the documents.

For endpoints which are not wrapped with `pnp-js-core` one can use [HttpClient](https://github.com/SharePoint/PnP-JS-Core/blob/master/src/net/httpclient.ts) or Fetch calls, like described [here](https://github.com/SharePoint/PnP-JS-Core/issues/536).

**Comment by patrick-rodgers** [comment]
What @koltyakov says is correct, there is no version information endpoint for items, only documents. Going to close this. Please _reopen_ should you require additional clarification. Thanks!

**Comment by RensBraspenning** [comment]
@patrick-rodgers I'm actually getting version information for list items and documents using the same endpoint. For example:

https://rensbraspenning.sharepoint.com/sites/verhuur/_api/web/lists/GetByTitle('Huuropzeggingen')/items/getbyid(6)/versions

**Comment by patrick-rodgers** [comment]
Interesting, I didn't think that worked last I checked. I'll reopen this and we can add the version part to item. Thanks!

**Comment by patrick-rodgers** [comment]
Ah, so taking a look this is possible but does not have the level of support as file versions (i.e. managing versions and restoring, etc). So we can add support for basic pulling of version info for an item, but there are no methods to do anything with that data.

**Comment by patrick-rodgers** [comment]
Merged in #651, will be part of 3.0.3
